### PR TITLE
Document theorems implementing CPCTC.

### PR DESCRIPTION
**Pull request by david-a-wheeler**
Many geometry textbooks refer to the theorem congruent triangles are congruent (CPCTC).  This commit adds notes to various comments to specifically indicate the various theorems that together constitute CPCTC.

In some sense this completes #49, which identifies CPCTC as an example of what it means to add "support for Euclidean Geometry".

Signed-off-by: David A. Wheeler <dwheeler@dwheeler.com>

### CODE REVIEW

**Review by jkingdon — APPROVED** [review]

File: set.mm [hunk 1]
```diff
@@ -382312,20 +382312,23 @@ proved using the inner form (see theorem 9.6 of [Schwabhauser] p. 69
       tgbtwnxfr.e $e |- ( ph -> E e. P ) $.
       tgbtwnxfr.f $e |- ( ph -> F e. P ) $.
       tgbtwnxfr.2 $e |- ( ph -> <" A B C "> .~ <" D E F "> ) $.
-      $( Deduce segment congruence from a triangle congruence.  (Contributed by
-         Thierry Arnoux, 27-Apr-2019.) $)
+      $( Deduce segment congruence from a triangle congruence.  This is a
+         segment portion of congruent triangles are congruent (CPCTC).
```
Comment (benjub) [inline set.mm:382316]: The second sentence seems to lack some words or have an extra verb ?
What does CPCTC stand for ?

Comment (david-a-wheeler) [reply set.mm:382316]: You're right. Sorry about that. I will tweak it later.

Comment (david-a-wheeler) [reply set.mm]: I've tweaked it, hopefully that resolves the problem!

**Comment by david-a-wheeler** [comment]
Thanks to @benjub for noting the poor wording, I think it's fixed now.

In the process I've realized that there are two variants of CPCTC not directly proved, namely, the other two corresponding angles of congruent triangles. We probably should add those theorems, but that's a different problem than just documenting common alternate names for theorems :-).

**Comment by benjub** [comment]
Thanks.  Now I can make sense of the comments.  Basically, I understand the first sentence of each comment, and I can agree that the second sentence is a rephrasing of the first.  I had never heard of "CPCTC" but if that's a staple of the US curriculum and you think adding it may help some people understand (or more likely, find) it better, then go for it.

**Comment by david-a-wheeler** [comment]
Your guess is exactly correct, CPCTC is a staple of many US curricula, which is why I want to make it clear where it's proved. Use a search engine on "geometry CPCTC" and you'll see it all over. I want to try to connect our proofs to terms widely used by others, so that we can connect our rigorous results with what others already know.

**Comment by jkingdon** [comment]
> Your guess is exactly correct, CPCTC is a staple of many US curricula, which is why I want to make it clear where it's proved. Use a search engine on "geometry CPCTC" and you'll see it all over.

I tried this and I found a whole lot more hits for "geometry SAS" or "geometry SSS". At least from memory, my (US) schooling included those terms but not CPCTC (at least not as an acronym).

>  I want to try to connect our proofs to terms widely used by others, so that we can connect our rigorous results with what others already know.

Totally agreed with this as a goal, although figuring out what terminology people know it by can sometimes be a bit tricky. Still, we should try as best we can.

**Comment by david-a-wheeler** [comment]
> I tried this and I found a whole lot more hits for "geometry SAS" or "geometry SSS". At least from memory, my (US) schooling included those terms but not CPCTC (at least not as an acronym).

SAS & SSS go the other way.

CPCTC starts with 2 congruent triangles, and lets you prove statements about segments and angles.  SAS & SSS start with segments (and an angle for SAS), and let you prove a statement that certain triangles are congruent.

**Review by benjub — APPROVED** [review]

File: set.mm [hunk 2]
```diff
@@ -382312,19 +382312,23 @@ proved using the inner form (see theorem 9.6 of [Schwabhauser] p. 69
       tgbtwnxfr.e $e |- ( ph -> E e. P ) $.
       tgbtwnxfr.f $e |- ( ph -> F e. P ) $.
       tgbtwnxfr.2 $e |- ( ph -> <" A B C "> .~ <" D E F "> ) $.
-      $( Deduce segment congruence from a triangle congruence.  (Contributed by
+      $( Deduce segment congruence from a triangle congruence.  This is a
+         portion of the theorem that congruent parts of congruent triangles are
+         congruent (CPCTC), focusing on a specific segment.  (Contributed by
```
Comment (avekens) [inline set.mm:382317]: CPCTC stands for "corresponding parts of congruent triangles are congruent" (see https://www.cuemath.com/geometry/cpctc/), not "congruent parts of ...".

**Review by avekens — CHANGES_REQUESTED** [review]
see inline comment

File: set.mm (hunk 2 above)
Comment (david-a-wheeler) [reply set.mm:382317]: You're absolutely correct, nice catch. I've fixed the change. I'll separately propose changes to other sections based on #2994, #2984, and #3014.

**Comment by david-a-wheeler** [comment]
I think this PR and https://github.com/metamath/set.mm/pull/3028 will clarify how we handle geometric congruence. My thanks to everyone who pitched in with relevant proofs!!

**Comment by david-a-wheeler** [comment]
@avekens - I think it's resolved, do you agree?

**Review by avekens — APPROVED** [review]

**Pull request merged**